res/skins/LateNight/effect_unit.xml

```diff
@@ -93,11 +93,12 @@
                   <Text>ON</Text>
                 </State>
                 <Connection>
-                  <ConfigKey persist="true"><Variable name="group"/>,enabled</ConfigKey>
+                  <ConfigKey persist="true"></ConfigKey>
                   <ButtonState>LeftButton</ButtonState>
                 </Connection>
                 <Connection>
-                  <ConfigKey persist="true"><Variable name="group"/>,enabled</ConfigKey>
+                  <!-- FIXME: Quick hack until effect chains are implemented. -->
```

Comment:
I don't think that would be worth the trouble considering the whole file will have to be replaced imminently. This is just a quick hack to make it keep working like it has been.